pkg/provisioner/controller.go

```diff
@@ -322,49 +322,35 @@ func (c *Controller) handleProvisionClaim(key string, obc *v1alpha1.ObjectBucket
 	return nil
 }
 
+// Delete or Revoke access to bucket defined by passed-in key.
 func (c *Controller) handleDeleteClaim(key string) error {
+	// Call `Delete` for new (greenfield) buckets with reclaimPolicy == "Delete".
+	// Call `Revoke` for new buckets with reclaimPolicy != "Delete".
+	// Call `Revoke` for existing (brownfield) buckets regardless of reclaimPolicy.
 
 	// TODO each delete should retry a few times to mitigate intermittent errors
+	// (Jon, is this TODO still needed??)
```

Comment:
@copejon ?

Reply:
yes, at the moment, the Delete() calls are a once off